code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -194,6 +194,21 @@
 		return access
 	return access | guest_pass.GetAccess()
 
+/obj/item/card/id/proc/attach_guest_pass(obj/item/card/id/guest/G, mob/user)
+	if(world.time > G.expiration_time)
+		to_chat(user, "There's no point, the guest pass has expired.")
```

Comment:
These could use spans

Reply:
any visual fix beyond minor grammatical errors feel out of scope